Dockerfile

```diff
@@ -39,6 +39,9 @@ WORKDIR /package-registry
 # Get in config which expects packages in /packages
 COPY config.docker.yml /package-registry/config.yml
 
+# Run as non-root user
+USER 1000
```

Comment:
I see some recommendations on setting UIDs > 10000 to avoid matching ids of local users. For example Chainguard uses UID 65532.

Do you think we could use a different UID here?

Reply:
> Well, regarding to obscurity I don't see it any different to setting it to 1000 🙂

that's why I said still valuable. That said, I think we both agree that the original image running with UID 0 isn't secure